src/widgets/image_widget.py

```diff
@@ -118,6 +123,7 @@ def draw_catalogue(self, ra_coords, dec_coords, size, colour_outline, colour_fil
 
     def reset_catalogue(self):
         self.catalogue_set = Render.reset_catalogue(self.catalogue_set)
+        self.canvas.draw()
```

Comment:
There's a decent chance I accidentally deleted this when I was merging all the render branches in... oops!